apps/frontend/src/components/card/MapPreview.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Trash, Delete, MapIcon } from 'kepler.gl/dist/components/common/icons';
+import { IconButton, IconTextButton } from '../buttons';
+
+interface MapPreviewProps {
+  status: boolean;
```

Comment:
A boolean status property doesn't mean anything.
You should instead reuse definitions define earlier and now available in the code ! (see `src/shared/models`)
Also if you thought about the `published` property of a project, I made a last minute change in #54 an it's now renamed to `draft`.